src/test/resources/log4j.properties

```diff
@@ -10,7 +10,7 @@ log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p [%t]
 log4j.logger.org.reflections=WARN
 
 # Show our logs
-log4j.logger.org.openstreetmap.atlas=TRACE
+log4j.logger.org.openstreetmap.atlas=DEBUG
```

Comment:
Any reason not to have Trace for tests?

Reply:
I found that the ChangeAtlas TRACE logs were... excessive. I could turn it off just for that package, I guess?